apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -119,7 +119,15 @@ const ManagedEventTypeDialog = dynamic(() => import("@components/eventtype/Manag
 
 const AssignmentWarningDialog = dynamic(() => import("@components/eventtype/AssignmentWarningDialog"));
 
-export type Host = { isFixed: boolean; userId: number; priority: number };
+export type Host = {
+  isFixed: boolean;
+  userId: number;
+  priority: number;
+  scheduleId: number | null;
+  availability?: AvailabilityOption | null;
```

Comment:
Why are you storing AvailabilityOption? We are already fetching all schedules of the user, you can find selected Option by matching it against scheduleId.

Reply:
This is required to remember the user selection for scenario when user is toggling between `Choose a common schedule`. When common schedule is choosen scheduleId is made null.